src/com/pentalog/pentastagiu/homework4/ListOfCircles.java

```diff
@@ -0,0 +1,26 @@
+package com.pentalog.pentastagiu.homework4;
+
+import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ListOfCircles {
+
+    public static void main(String[] args) {
+        double PI = Math.PI;
+        List<Object> circles = new ArrayList<>();
```

Comment:
Nu e bine sa iti faci liste asa generice cand stii ce date o sa ai in Lista, si mai ales daca ai un singur tip